crates/orchestrator/src/tests/jobs/state_update_job/mod.rs

```diff
@@ -0,0 +1,152 @@
+use mockall::predicate::eq;
+use rstest::*;
+use settlement_client_interface::MockSettlementClient;
+
+use std::{collections::HashMap, fs};
+
+use super::super::common::init_config;
+
+use crate::jobs::{
+    constants::{
+        JOB_METADATA_STATE_UPDATE_BLOCKS_TO_SETTLE_KEY, JOB_METADATA_STATE_UPDATE_FETCH_FROM_TESTS,
+        JOB_PROCESS_ATTEMPT_METADATA_KEY,
+    },
+    state_update_job::StateUpdateJob,
+    types::{JobStatus, JobType},
+    Job,
+};
+
+use httpmock::prelude::*;
+
+#[rstest]
+#[tokio::test]
+async fn test_create_job() {
```

Comment:
can we add a test case for retries as well? feel free to create an issue if you want it to be addresses in a new PR

Reply:
Created an issue here:
https://github.com/karnotxyz/madara-orchestrator/issues/30